cmd/controller-manager/app/controllermanager.go

```diff
@@ -255,6 +255,7 @@ func startClusterController(ctx controllerscontext.Context) (enabled bool, err e
 		EnableTaintManager:                 ctx.Opts.EnableTaintManager,
 		ClusterTaintEvictionRetryFrequency: 10 * time.Second,
 		ExecutionSpaceRetryFrequency:       10 * time.Second,
+		RateLimiterOptions:                 ctx.Opts.RateLimiterOptions,
```

Comment:
The following controllers are missing settings
`startApplicationFailoverController`, `startEndpointSliceCollectController`, `startEndpointSliceDispatchController`, `startServiceImportController`, `startUnifiedAuthController`, `startWorkloadRebalancerController`, `startAgentCSRApprovingController`,